mssql.go

```diff
@@ -577,6 +577,9 @@ func (s *Stmt) makeRPCParams(args []namedValue, isProc bool) ([]param, []string,
 		}
 		params[i+offset].Name = name
 		decls[i] = fmt.Sprintf("%s %s", name, makeDecl(params[i+offset].ti))
```

Comment:
I would prefer to compose the delcs in a single Sprintf, rather then adding an optional output. Maybe do something like this:
```
const outputSuffix = " output"
var output string
if isOutputValue(val.Value) {
    output = outputSuffix
}
decls[i] = fmt.Sprintf("%s %s%s", name, makeDecl(params[i+offset].ti), output)

```

Reply:
Where would you declare the const? At the top of mssql.go?

Reply:
just inline in the func itself. Go will treat it the same anywhere you declare it.